dependabot/utils.py

```diff
@@ -88,7 +88,7 @@ def commit_file(repository_name, file_path, updated_file_content, commit_branch,
             remote_file_in_pr_branch = ""
 
         if updated_file_content == remote_file_contents:
-            return False
+            return ""
```

Comment:
Any reason this util function was changed?

Reply:
The commit can be used to notify the lag color changes if there are any, so I updated this to return the commit.sha instead of returning a True/False

Reply:
Its a common utils function used in all 3 scripts. HAve you followed up so that they don't break?

IMO it better to return 2 params in this case: commit_file returns is_updated, sha

Reply:
It does not break since if a string is checked against an if condition it returns false if its empty and true otherwise. But yes your suggestion is better.